frontend/public/actions/features.ts

```diff
@@ -189,6 +189,9 @@ const ssarChecks = [{
 }, {
   flag: FLAGS.CAN_LIST_OPERATOR_GROUP,
   resourceAttributes:{ group: OperatorGroupModel.apiGroup, resource: OperatorGroupModel.plural, verb: 'list' },
+}, {
+  flag: FLAGS.CAN_LIST_CHARGEBACK_REPORTS,
+  resourceAttributes:{ resource: ChargebackReportModel.plural, metadata: {namespace: 'openshift-metering'}, verb: 'list' },
```

Comment:
viewing results actually requires separate permissions from list reports: https://github.com/operator-framework/operator-metering/blob/57c8e902324bfc70e65b635a48f2ffc93506900b/charts/openshift-metering/values.yaml#L426

Reply:
I would suggest hiding `ReportData` on the details page if user doesn't have permission, but still let them list reports if they can.

Reply:
> I would suggest hiding `ReportData` on the details page if user doesn't have permission, but still let them list reports if they can.

On the main Chargeback listing page, I would need to disable [Create Report] and most of the kabob menu items such as 'Edit Labels, ...Annotations, Delete Report' , etc.., correct?  Before they even drill down into a details page.

Reply:
> viewing results actually requires separate permissions from list reports:

Hi, I added the `group` to the attributes:

`resourceAttributes:{ group: 'metering.openshift.io', resource: 'reports', metadata: {namespace: 'openshift-metering'}, verb: 'list' },`

Is this correct?  -thanks

Reply:
> On the main Chargeback listing page, I would need to disable [Create Report] and most of the kabob menu items such as 'Edit Labels, ...Annotations, Delete Report' , etc.., correct? Before they even drill down into a details page.

I believe this will work already without any special code. We should be handling this generally for all resources.

Reply:
Yes, it definitely needs the group! Missed that on review, thanks.

Reply:
> > On the main Chargeback listing page, I would need to disable [Create Report] and most of the kabob menu items such as 'Edit Labels, ...Annotations, Delete Report' , etc.., correct? Before they even drill down into a details page.
> 
> I believe this will work already without any special code. We should be handling this generally for all resources.

Ok, how do I create a user that can list chargeback reports, but not edit them?  -thanks

Reply:
I'd suggest assigning a user the existing cluster-reader role.

Reply:
If that doesn't work, you'd probably need to create a custom role.

Reply:
https://github.com/operator-framework/operator-metering/blob/master/manifests/metering-config/advanced-auth.yaml#L6-L35 may be useful

Reply:
Fixed/addressed